BVH.cs

```diff
@@ -225,6 +279,12 @@ public void RenderDebug()
                         enableInstancing = true
                     };
                 }
+
+				int iterations = (int)Mathf.Floor((float)matricies.Count / 1023.0f);
+				for (int i = 0; i < iterations; i++) {
+					Graphics.DrawMeshInstanced(mesh, 0, _debugRenderMaterial, matricies.GetRange(0, 1023));
+					matricies.RemoveRange(0, 1023);
+				}
                 Graphics.DrawMeshInstanced(mesh, 0, _debugRenderMaterial, matricies);
```

Comment:
Should this not be removed after adding the split draw or are both calls necessary?

Reply:
Both calls would be necessary as the for-loop should only execute when the limit is exceeded, which often occurs when each triangle in a scene has its own AABB.